src/components/Login.js

```diff
@@ -15,7 +15,32 @@ function Login() {
                 password,
             })
         })
-        alert('Successful login!')
+        const msg = await resp.text()
+        if ( msg === 'ok') {
+            alert('Successful login!')
+        } else if ( msg == 'Wrong username or password') {
+            alert('Unsuccessful login! 😕')
+        }
+        
+    }
+
+    async function handleLogout() {
+        const resp = await fetch('/logout', {
+            method: 'DELETE',
```

Comment:
Somewhere it was recommended to use `DELETE` for logout, but it's not mandatory. No body is needed for this fetch.